INSTALL.md

```diff
@@ -1,10 +1,11 @@
 # Install Zync
```

Comment:
What are the prerequisites to install zync? 
We usually have these sections: prerequisites and procedure.
If necessary, you can add a reference section to include additional information.